app/routes/ready-reckoner/land-values.js

```diff
@@ -53,6 +53,21 @@ function addState (input) {
   }, {})
 }
 
+const standardsSetCalculations = [
```

Comment:
Potentially data that would need to come from an API i.e. with management capability. I've left this here until we do the larger refactoring.